mypyc/lib-rt/CPy.h

```diff
@@ -64,6 +64,14 @@ typedef struct tuple_T4CIOO {
 } tuple_T4CIOO;
 #endif
 
+// System-wide empty tuple constant
+extern PyObject * __mypyc_empty_tuple__;
+
+static inline PyObject *_CPyTuple_LoadEmptyTupleConstant() {
```

Comment:
There's no need for the `_` prefix. Public functions have the `CPy` prefix, even if they are static inline.